draft-demarco-oauth-status-attestations.md

```diff
@@ -175,17 +175,19 @@ The general requirements for the implementation of Status Assertion are
 listed in this section. The Status Assertion:
 
 - SHOULD be presented in conjunction with the Digital Credential.
-The Status Assertion MUST be timestamped with its issuance datetime,
-using a timestamp which is later then the time of presentation issuance;
-- MUST contain the expiration datetime after which the Digital Credential
+- MUST include information that links it to the
+referenced Digital Credential;
+- MUST be timestamped with its issuance datetime,
+using a timestamp which is at or after the time of
+Digital Credential which it refers;
+- MUST contain the expiration datetime after which both
+the Status Assertion and the Digital Credential it refers
```

Comment:
I might leave off the validity alignment that's implied here. The validity of the assertion and the validity of the credential could be subject to different verifier policies.... There might be multiple status assertion for a single credential, and a policy would accept m of n.

Reply:
Very good point. Moved to an issue for future evaluations